conftest.py

```diff
@@ -59,3 +60,47 @@ def profile3(user3):
 def professional(profile3):
     return Professional.objects.create(profile_id=profile3, profession=Professions.Locksmith,
                                        description='I')
+
+
+@pytest.fixture
+def typeOfJob(professional):
+    return TypeOfJob(professional_id=professional, typeOfJob_name="Gel nail polish", price=90)
+
+
+@pytest.fixture
+def persisted_typeOfJob(typeOfJob):
+    typeOfJob.professional_id.save()
+    typeOfJob.save()
+    return typeOfJob
+
+
+@pytest.fixture
+def appointment(typeOfJob, professional, client):
+    return Appointment(professional_id=professional, client_id=client, typeOfJob_id=typeOfJob,
```

Comment:
Do you think,
That those fixures needed by other apps? 
In case not, maintain it under the relevant app

Reply:
Done